src/lib/types.ts

```diff
@@ -42,18 +42,6 @@ export function instanceOfCommand(object: any): object is Command {
   return 'title' in object && 'command' in object;
 }
 
-export interface HubLValidationError {
-  reason: string;
-  message: string;
-  lineno: number;
-  startPosition: number;
-  categoryErrors: {
-    fullName?: string;
-    path?: string;
-  };
-  category: string;
-}
-
```

Comment:
This type exists in `local-dev-lib` now, so changing our usage to that one and removing this. Theres some others I'm sure that we could swap to LDL types but this is just one I specifically noted